event_track_generate/views/event.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+     © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. -->
+
+<openerp>
+<data>
+
+<act_window
+    id="generator_action"
+    name="Generate event tracks"
+    src_model="event.event"
+    res_model="event_track_generate.generator"
+    view_mode="form"
+    target="new"
+    key2="client_action_multi"/>
+
+<record id="generator_view_form" model="ir.ui.view">
+    <field name="name">Event track generator</field>
+    <field name="model">event_track_generate.generator</field>
+    <field name="arch" type="xml">
+        <form>
+            <group name="main">
+                <group string="Dates and event data">
+                    <field name="event_id" readonly="True"/>
+                    <field name="event_date_begin" readonly="True"/>
+                    <field name="event_date_end" readonly="True"/>
+                    <field name="event_date_tz" readonly="True"/>
+                    <field name="name"/>
+                    <field name="location_id"/>
+                    <field name="speaker_ids" widget="many2many_tags"/>
+                    <field name="tag_ids" widget="many2many_tags"/>
+                </group>
+                <group string="Weekdays">
+                    <field name="mondays"/>
+                    <field name="tuesdays"/>
+                    <field name="wednesdays"/>
+                    <field name="thursdays"/>
+                    <field name="fridays"/>
+                    <field name="saturdays"/>
+                    <field name="sundays"/>
+                </group>
+                <group string="Schedule">
+                    <field name="start_time" widget="float_time"/>
+                    <field name="duration" widget="float_time"/>
+                    <field name="end_time" widget="float_time"/>
+                </group>
+                <group string="Other options">
+                    <field name="delete_existing_tracks"/>
+                    <field name="publish_tracks_in_website"/>
+                    <field name="adjust_start_time"/>
+                    <field name="adjust_end_time"/>
+                </group>
+            </group>
+            <footer>
+                <button name="action_generate"
+                        type="object"
+                        string="Generate tracks"
+                        class="oe_highlight"/>
+                or
```

Comment:
I'm not talking about the or, but the last line in translation file that it's commented